psi4/driver/driver.py

```diff
@@ -1706,10 +1703,6 @@ def frequency(name, **kwargs):
     for postcallback in hooks['frequency']['post']:
         postcallback(lowername, wfn=wfn, **kwargs)
 
-    # Reset old global basis if needed
-    if not old_global_basis is None:
-        core.set_global_option("BASIS", old_global_basis)
```

Comment:
Nice to get this old inconsistency of `old_global_basis` and `lowername, new_basis = lowername.split('/')` out of here and put freq on par with gradient.